driver/src/test/java/org/neo4j/driver/v1/util/cc/ClusterRule.java

```diff
@@ -52,11 +53,28 @@ public AuthToken getDefaultAuthToken()
         return AuthTokens.basic( "neo4j", PASSWORD );
     }
 
+    public static void stopSharedCluster()
+    {
+        if ( SharedCluster.exists() )
+        {
+            try
+            {
+                SharedCluster.stop();
+            }
+            finally
+            {
+                SharedCluster.remove();
+            }
+        }
+    }
+
     @Override
     protected void before() throws Throwable
     {
         assumeTrue( "BoltKit cluster support unavailable", boltKitAvailable() );
 
+        stopSingleInstanceDatabase();
```

Comment:
Do we still have problems to have both single instance and shared cluster running at the same time? This change might make test slower by stopping and restarting server additionally in our tests, right?

Reply:
Running both cluster and single instance db leaves only a small amount of free memory on the machine, this makes PR build occasionally hit OOM. I also thought it would make builds significantly slower but it did not, last 3 builds for this change have very similar time as other builds.